src/fundamentals/xcm_executor.rs

```diff
@@ -34,12 +36,34 @@ pub trait XcmConfig {
 }
```

Comment:
Any reason for exposing `AssetTransactor` and `Barrier` to them in these fundamentals?

I could only make it work by removing the `start` feature and using the `example` feature for using the available `AssetTransactor` there.
Do we want them to implement their own at this stage? If that's the case, I'll probably just create a new simpler trait with only `withdraw` and `deposit`.

I think a good aspect of this "fundamentals" section is we tackle things one at a time, so maybe not a good idea to put the `Barrier` here already. That could go in a separate lesson on setting up the barrier and only the barrier.

Reply:
Maybe they should just tackle this after the fundamentals, and we should stick to the simplest version of the executor here.

Reply:
>  Do we want them to implement their own at this stage? If that's the case, I'll probably just create a new simpler trait with only withdraw and deposit.

Yes, i think something like this